mafContrib/updatedDitherStackers.py

```diff
@@ -0,0 +1,1196 @@
+##############################################################################################################
+# Motivation: implement various dithering strategies.
+
+# Updated
+# Better repulsive random algorithm in terms of execution style
+# New season stacker
+
+# Some of the stackers here are modified versions of the stackers in lsst.sims.maf.stackers.ditherStackers.
+# These function similarly to ditherStackers,  with the major difference that dithers here are restricted to
+# within the hexagons used to tile the sky. Each of these hexagons is inscribed in a FOV, and having dithers
+# restricted to within these hexagons allows avoiding oversampling near the boundaries of the hexagonal tiles.
+ 
+# Also, the stackers here follow new naming scheme:  [Pattern]Dither[Field]Per[Timescale]. The absence of the
+# keyword 'Field' implies dither assignment to all fields.
+
+# Humna Awan: humna.awan@rutgers.edu
+# Date last modified: 04/09/16
+###############################################################################################################
+import numpy as np
+import matplotlib.pyplot as plt
+from lsst.sims.maf.stackers import BaseStacker
+from mafContrib.SeasonStacker_v2 import SeasonStacker_v2 as SeasonStacker
```

Comment:
This is now a stacker in core maf. Please import from there. (does not exist in mafContrib anymore) https://github.com/lsst/sims_maf/blob/master/python/lsst/sims/maf/stackers/generalStackers.py